mappings/net/minecraft/block/entity/BlockEntity.mapping

```diff
@@ -42,6 +44,14 @@ CLASS net/minecraft/class_2586 net/minecraft/block/entity/BlockEntity
 		ARG 2 state
 	METHOD method_31664 setCachedState (Lnet/minecraft/class_2680;)V
 		ARG 1 state
+	METHOD method_38235 toUpdatePacket ()Lnet/minecraft/class_2596;
+		COMMENT Implement and return a packet that should be sent to players nearby when the observable state of
```

Comment:
A question: in code, this is `@Nullable` and defaults to return `null`. Probably worth to note the `null` behavior, e.g. if it is not sent at all or sent with some other default mechanism.